src/Entities/Transformers/Family.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Choredo\Entities\Transformers;
+
+use Assert\Assertion;
+use Choredo\Entities;
+use League\Fractal\TransformerAbstract;
+
+class Family extends TransformerAbstract
+{
+    const DAYS_OF_WEEK = ['sunday', 'monday', 'tuesday', 'wednesday', 'thursday', 'friday', 'saturday'];
+
+    public function transform(Entities\Family $family)
+    {
+        return [
+            'type'       => 'family',
```

Comment:
If you use the JsonApi serializer, this isn't necessary. It'll pull the type, ID and links for you automagically. you just return an array.

Reply:
:+1: